config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  resources :sessions
```

Comment:
your `sessions` resource should only cover three routes `:new`,`:create`,`:destroy`.